### ISSUE TITLE
[Accessibility] Request body edit box is not accessible via keyboard.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1858]
**Test Environment:**
OS Build: Version 22H2(OS Build 25140.1000) 
Browser: Version 104.0.1278.2 (Official build) dev (64-bit)
URL: [Graph Explorer | Try Microsoft Graph APIs - Microsoft Graph](https://developer.microsoft.com/en-us/graph/graph-explorer)

**Repro Steps:**

Open the above [URL](https://developer.microsoft.com/en-us/graph/graph-explorer) in Edge browser.
Navigate to the Request body edit box using tab key.
Observe the issue.

**Actual Result:**

Request body edit box is not accessible via keyboard.

**Expected Result:**
Request body edit box should be accessible via keyboard.
As the user navigates the menu item request body etc. With the tab key the keyboard focus should next land on the request body.

**User Impact:** 
If the control is in disabled state, the user should not be able to activate it with mouse or keyboard.

**MAS reference:**
[Keyboard - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.01.01)
